ManagerRegistry.php

```diff
@@ -16,9 +16,25 @@
 
 use Doctrine\ODM\MongoDB\MongoDBException;
 use Symfony\Bridge\Doctrine\ManagerRegistry as BaseManagerRegistry;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 class ManagerRegistry extends BaseManagerRegistry
 {
+    public function __construct(ContainerInterface $container, $name, array $connections, array $managers, $defaultConnection, $defaultManager, $proxyInterfaceName)
```

Comment:
Isn’t this signature change a BC break, requiring a major release?

Reply:
Hm yes I guess it would break if someone relies on arguments index to override the definition. Could we rely on setter injection instead?

Reply:
You can add the `ContainerInterface` at the end and change its default value to `null`. That way it's not a BC break.

Reply:
Oh right